minkindr/include/kindr/minimal/implementation/rotation-quaternion-inl.h

```diff
@@ -472,13 +496,24 @@ RotationQuaternionTemplate<Scalar>::log() const {
 }
 
 template<typename Scalar>
-bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(const RotationMatrix& matrix) {
-  const Scalar kThreshold = static_cast<Scalar>(1.0e-7);
-  if (std::fabs(matrix.determinant() - static_cast<Scalar>(1.0)) > kThreshold) {
+bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(
+    const RotationMatrix& matrix) {
+  constexpr Scalar kThreshold = static_cast<Scalar>(1.0e-8);
+  return isValidRotationMatrix(matrix, kThreshold);
+}
+
+template<typename Scalar>
+bool RotationQuaternionTemplate<Scalar>::isValidRotationMatrix(
+    const RotationMatrix& matrix, const Scalar threshold) {
+  if (std::fabs(matrix.determinant() - static_cast<Scalar>(1.0)) > threshold) {
+    VLOG(5) << matrix.determinant();
+    VLOG(5) << matrix.determinant() - static_cast<Scalar>(1.0);
     return false;
   }
-  if ((matrix * matrix.transpose() - RotationMatrix::Identity()).cwiseAbs().maxCoeff()
-      > kThreshold) {
+  if ((matrix * matrix.transpose() -
+      RotationMatrix::Identity()).cwiseAbs().maxCoeff() > threshold) {
+    VLOG(5) << matrix * matrix.transpose();
+    VLOG(5) << matrix * matrix.transpose() - RotationMatrix::Identity();
```

Comment:
can we put this into a higher log level (maybe 200), so that these levels are available for higher level debugging?